releasenotes/notes/bump-horizon-694d426decbf7df3.yaml

```diff
@@ -0,0 +1,5 @@
+---
+security:
+  - |
+    Update Horizon on Ubuntu to include apache2 patch `2.4.52-1ubuntu4.7`
```

Comment:
Remember to use double quotes to format text with monospace font. Single quote generates italics.